src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -102,31 +86,26 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
         <div />
 
         <div className="PosterPage__header--middle-cell">
-          <div className="PosterPage__headerInfo">
-            {posterId && <div className="PosterPage__posterId">{posterId}</div>}
-            {moreInfoUrl && (
+          <p className="PosterPage__title">{title}</p>
+          <div className="PosterPage__authorBox">
+            Presented by: {presenterName}
+          </div>
+          <div className="PosterPage__description">{description}</div>
+          <div className="PosterPage__descriptionSecondary">
```

Comment:
rename or `@debt`?

Reply:
Neither, they're the right name. Even if they're not ideal.